unittest/srdf.cpp

```diff
@@ -60,6 +60,38 @@ BOOST_AUTO_TEST_CASE(readReferenceConfig)
   BOOST_CHECK(!q2.isZero());
 
   
+}
+  
+BOOST_AUTO_TEST_CASE(readReferenceConfig_stream)
+{
+  const string urdf =
+    "<robot name='test'>"
+    "<link name='base_link'/>"
+    "<link name='child_link'/>"
+    "<joint type='revolute' name='joint'>"
+    "  <parent link='base_link'/>"
+    "  <child link='child_link'/>"
+    "  <limit effort='30' velocity='1.0' />"
+    "</joint>"
+    "</robot>";
+  const string srdf =
+    "<robot name='test'>"
+    "<group_state name='reference' group='all'>"
+    "     <joint name='joint'  value='0.0' />"
+    "</group_state>"
+    "</robot>";
+  
+  Model model;
+  pinocchio::urdf::buildModelFromXML(urdf, model);
+
+  std::istringstream iss (srdf);
+  pinocchio::srdf::loadReferenceConfigurations(model,iss,false);
```

Comment:
This was failing before because unbounded revolute joints where not handled properly.